index.js

```diff
@@ -3,6 +3,8 @@ var wiredep = require('wiredep').stream;
 var elixir = require('laravel-elixir');
 var path = require('path');
 var task = elixir.Task;
+var econ = elixir.config;
+var merge = require('merge-stream');
```

Comment:
I didn't like the idea of adding a new dependency to the  project, there's no need